CITATION.cff

```diff
@@ -4,7 +4,7 @@ message: "Please use this information to cite ONNX Runtime in
   research or other publications."
 authors:
   - affiliation: Microsoft Corporation
```

Comment:
Remove the affiliation according to https://citation-file-format.github.io/assets/pdf/cff-specifications-1.0.0.pdf:

<img width="227" alt="image" src="https://github.com/microsoft/onnxruntime/assets/30328909/c03f6d2b-fc5f-4922-afda-8d15aafe221c">